src/main/java/controller/UserLoginController.java

```diff
@@ -0,0 +1,48 @@
+package controller;
+
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+import exception.UnauthorizedRequestException;
+import http.request.HttpRequest;
+import http.response.HttpResponse;
+import http.response.HttpResponseGenerator;
+import service.UserService;
+import utils.FileIoUtils;
+import utils.ResourcePathUtils;
+
+public class UserLoginController extends AbstractController {
+	@Override
+	public void doPost(HttpRequest httpRequest, DataOutputStream dos) {
+		UserService userService = new UserService();
+
+		boolean result = userService.login(httpRequest.getHttpRequestBody().get("userId"),
```

Comment:
한 줄에 너무 많은 내용을 담고 있어 코드 읽기가 힘드네요
가독성을 위해 변수에 담아보는게 어떨까요?
```suggestion
		final Map<String, String> requestBody = httpRequest.getHttpRequestBody();
		final String userId = requestBody.get("userId");
		final String password = requestBody.get("password");

		boolean result = userService.login(userId, password);
```

Reply:
수정했습니다~! 감사합니다 :)